docs/source/conf.py

```diff
@@ -363,7 +363,7 @@
     ],
     "show_version_warning_banner": True,
     "switcher": {
-        "json_url": "/docs/_static/versions.json",
+        "json_url": "https://arrow.apache.org/docs/_static/versions.json",
```

Comment:
Is there another way to do this that doesn't depend on an internet connection?

Reply:
For local development we could fall back to `docs/source/_static/versions.json` and maybe we could add an environment variable that can be set when developing locally and in case of the preview build.

I can add that changes today. If this PR needs to get into 21.0.0 I can also do move `json_url` work into a separate PR.

Reply:
Fine with me to do the work in another PR. Also fine if we want to leave it as you have it either way.